sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,308 @@
+"""
+Calculate SDO/AIA wavelength and temperature response functions.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import os
+import warnings
+from collections import namedtuple
+
+import numpy as np
+import h5py
```

Comment:
Would it be possible / worth it to make h5py optional on import? We could just not cache the results if h5py is not installed right?